man/systemd.link.xml

```diff
@@ -410,7 +410,7 @@
         </listitem>
       </varlistentry>
       <varlistentry>
-        <term><varname>TxQueueLength=</varname></term>
+        <term><varname>TransmitQueueLength=</varname></term>
```

Comment:
"Tx" is one of those abbreviations i'd actually be OK with. It's very very standard I'd say. But Transmit is good too, and internal consistency matters most I guess

Reply:
Yeah, but we already have e.g. `TransmitChecksumOffload=`. Just for consistency.